internal/cmd/service/start/controller/start_controller_cmd.go

```
@@ -44,7 +44,7 @@ import (
 	"github.com/osac-project/fulfillment-service/internal/controllers/baremetalinstance"
 	"github.com/osac-project/fulfillment-service/internal/controllers/cluster"
 	"github.com/osac-project/fulfillment-service/internal/controllers/computeinstance"
-	"github.com/osac-project/fulfillment-service/internal/controllers/organization"
+	"github.com/osac-project/fulfillment-service/internal/controllers/tenant"
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofmt)